/*
 * This file was automatically generated by EvoSuite
 * Thu Mar 10 09:49:00 GMT 2022
 */

package failing;

import org.junit.Test;
import static org.junit.Assert.*;
import static org.evosuite.runtime.EvoAssertions.*;
import failing.Failing;
import java.io.IOException;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true, useJEE = true) 
public class Failing_ESTest extends Failing_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      Failing failing0 = new Failing();
      try { 
        failing0.throwUncaughtException((String) null);
      } catch(NullPointerException e) {
         //
         // Throwing NullPointerException
         //
         verifyException("failing.Failing", e);
      }
  }

  @Test(timeout = 4000)
  public void test1()  throws Throwable  {
      Failing failing0 = new Failing();
      try { 
        failing0.throwCaughtException(805);
      } catch(IOException e) {
         //
         // Throwing IOException
         //
         verifyException("failing.Failing", e);
      }
  }
}
